gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/setup.dev.flow.ts

```diff
@@ -51,9 +51,9 @@ export default class SetupFlow extends FlowCommand<TransactionResponse> {
         '9n1sSGA5rhfsQyaX3tHz3ZU1ffR6V8KffvWtFPBcFrJw',
```

Comment:
I see this file is used as a means of debugging and we plan to remove it?
If it's useful, is there an option to restructure this as a test?

Reply:
That's exactly the plan, move it to a test once we have the framework